preview/example.js

```diff
@@ -1,5 +1,10 @@
+var fs = require('fs');
+var insertStyle = require('insert-css');
 
-b = require('boxbits');
+var css = fs.readFileSync(__dirname + '/reset.css', 'utf8');
```

Comment:
Resetting styling is a concern of the library itself, not the example code. I know that the index file had the `reset.css` linked, but that was intended to be a temporary build-out shim. I would be comfortable moving this `insertStyle` usage into the main lib coffee file though, it's a nice way to clean up the example HTML already!